Why, oh why would you point these to master, upstream...
--- a/data/org.gnome.Builder.appdata.xml.in
+++ b/data/org.gnome.Builder.appdata.xml.in
@@ -39,27 +39,27 @@
   <screenshots>
     <screenshot type="default">
       <caption>Quickly access your projects</caption>
-      <image width="1600" height="900">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/01-greeter.png</image>
+      <image width="2400" height="1350">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/01-greeter.png</image>
     </screenshot>
     <screenshot type="default">
       <caption>Create new projects from a variety of templates</caption>
-      <image width="1600" height="900">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/02-template.png</image>
+      <image width="2400" height="1350">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/02-template.png</image>
     </screenshot>
     <screenshot type="default">
       <caption>Integration with Git to clone your projects</caption>
-      <image width="1600" height="900">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/03-clone.png</image>
+      <image width="2400" height="1350">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/03-clone.png</image>
     </screenshot>
     <screenshot type="default">
       <caption>The source code editor</caption>
-      <image width="1600" height="900">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/04-editor.png</image>
+      <image width="2400" height="1350">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/04-editor.png</image>
     </screenshot>
     <screenshot type="default">
       <caption>Fast global fuzzy search</caption>
-      <image width="1600" height="900">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/05-search.png</image>
+      <image width="2400" height="1350">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/05-search.png</image>
     </screenshot>
     <screenshot type="default">
       <caption>Debug with the GNU debugger</caption>
-      <image width="1600" height="900">https://git.gnome.org/browse/gnome-builder/plain/data/appdata/06-debugger.png</image>
+      <image width="2400" height="1350">https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/data/appdata/06-debugger.png</image>
     </screenshot>
   </screenshots>
 

